<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QWidget" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width> <!-- Ширина как у админа для консистентности -->
    <height>450</height> <!-- Высота как у админа для консистентности -->
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мои файлы</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="searchLineEdit">
       <property name="placeholderText">
        <string>Поиск по имени файла...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uploadButton">
       <property name="enabled">
        <bool>true</bool> <!-- Сразу доступна -->
       </property>
       <property name="text">
        <string>Загрузить файл</string>
       </property>
        <property name="toolTip">
         <string>Загрузить новый файл на сервер</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="filesTableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool> <!-- Колонка действий не растягивается -->
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Имя файла</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Размер</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата загрузки</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Просмотры</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Действия</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="uploadProgressBar">
      <property name="visible">
       <bool>false</bool> <!-- Скрыт по умолчанию -->
      </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="format">
      <string>Загрузка: %p%</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
